tests/js/JSONFormatBearTest.py

```diff
@@ -25,6 +26,20 @@
 }"""
 
 
+class ParseExceptionTest(unittest.TestCase):
+
+    def test_parse_err_msg(self):
+        valid_msg = ['Expecting value: line 20 column 30 (char 45)',
+                     'Expecting value: line 1 column 1 ']
+        invalid_msg = ['Expecting property change',
+                       'Expecting line 20 column 30',
+                       'Expecting value: line column']
+        for msg in valid_msg:
+            self.assertTrue(JSONFormatBear.parse_err_msg(msg))
+        for msg in invalid_msg:
+            self.assertFalse(JSONFormatBear.parse_err_msg(msg))
```

Comment:
inlining means in this case to inline the content, not just the function ;)

> For tests, I can assert group(1), group(2), group(3) for the expected output. Would that be sufficient?

I think you should still assert for the bear results, as they get delivered to the user, and that's where we want to test ideally.